### ISSUE TITLE
incorrect property in drawRoundRect method causes improper drawing 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [paazmaya/papervision3d#194]
```
What steps will reproduce the problem?
1. Create a vectorShape3D Object
2. Call the drawRoundRect method of the graphics property of the vector
shape and pass it different values for the ellipse width and ellipse height.
3.

What is the expected output? What do you see instead?
I expect to see a rounded rect where the corners have the same curves - but
the upper right will appear differently from the other 3 corners.

What version of the product are you using? On what operating system?
version 910 -- Windows XP Sp3

Please provide any additional information below.

Basically the problem is an incorrect variable name in the drawRoundRect
method in the Graphics3D class. Line 65 reads:
curveTo(x + width, y + height , x + width - ellipseHeight, y + height);
but the third parameter should be
x + width - ellipseWidth
This is a problem that is easy to miss because most people create rounded
rects where the ellipseWidth and the ellipseHeight are the same value. It
should be a quick fix

```

Original issue reported on code.google.com by `MarkHMar...@gmail.com` on 19 May 2009 at 3:06
